common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4117,7 +4117,7 @@ public static enum ConfVars {
         "${system:java.io.tmpdir}" + File.separator + "${system:user.name}" + File.separator +
             "operation_logs",
         "Top level directory where operation logs are stored if logging functionality is enabled"),
-    HIVE_SERVER2_LOGGING_OPERATION_LEVEL("hive.server2.logging.operation.level", "EXECUTION",
+    HIVE_SERVER2_LOGGING_OPERATION_LEVEL("hive.server2.logging.operation.level", "NONE",
```

Comment:
wait, why change here, when you override on client side?

Reply:
We override only on Beeline because as we said Beeline is mostly used by developers.
It's for other Hive client tools, like Hue.

Reply:
are we disabling it for the JDBC connections?